src/hooks/useVideo.tsx

```diff
@@ -1,34 +0,0 @@
-import {useRef, useState, useEffect, useCallback} from 'react'
-import videojs from 'video.js'
-import 'video.js/dist/video-js.css'
-import * as React from 'react'
-
-export const useVideoJS = (videoJsOptions: any) => {
```

Comment:
This is the only place where `video.js` packages were being used. So removing this hook means we can also remove those three packages from `package.json`.